apps/cli/src/lib/compiler/comamnds.ts

```diff
@@ -0,0 +1,57 @@
+import glob from "glob";
```

Comment:
this file has a typo `comamnds.ts` instead of `commands.ts`

Reply:
resolved in [34c728a](https://github.com/Disploy/disploy/pull/19/commits/34c728a119668093bd8fe49b35ef4cc25cdbb366)